homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -426,7 +419,12 @@ def unique_id(self) -> str:
     @property
     def extra_state_attributes(self):
         """Return the state attributes."""
-        return {"device_id": self.device_id, "zone_id": self._device["zone"]}
+        attributes = {
+            "device_id": self.device_id,
+        }
+        if self._device.get("zone"):
+            attributes["zone_id"] = self._device["zone"]
```

Comment:
```suggestion
        if zone := self._device.get("zone"):
            attributes["zone_id"] = zone
```

slightly faster as it only does one lookup